Update Sidecar Docs with Latest Version (#471)
PureStake/moonbeam-docs: builders/get-started/eth-compare/tx-fees.md

=== "EIP-1559"
    ```

>     Gas Price = Base Fee + Max Priority Fee Per Gas< Max Fee Per Gas ? 

Reviewer: ```suggestion
    Gas Price = Base Fee + Max Priority Fee Per Gas < Max Fee Per Gas ? 
```

>     Gas Price = BaseFee + MaxPriorityFeePerGas < MaxFeePerGas ? 
>                 BaseFee + MaxPriorityFeePerGas : 
>                 MaxFeePerGas;
>     Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}

Reviewer: ```suggestion
    Gas Price = Base Fee + Max Priority Fee Per Gas < Max Fee Per Gas ? 
                Base Fee + Max Priority Fee Per Gas : 
                Max Fee Per Gas;
    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
```

all of the other variables have a space between the words so I would change these just to be consistent

    ```
=== "Legacy"
    ```
    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
    ```
=== "EIP-2930"
    ```
    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
    ```

With the introduction of RT1900, there is a `Transaction Weight` mismatch between what is reported by the Sidecar API and what is used for the EVM transaction fee. Consequently, you need to add the following amount to `Transaction Weight`:

=== "Moonbeam"
    ```
    86298000
    ```
=== "Moonriver"
    ```
    86298000
    ```
=== "Moonbase Alpha"
    ```
    250000000
    ```

> The values of `Gas Price`, `MaxFeePerGas` and `MaxPriorityFeePerGas` for the applicable transaction types can be read from the block JSON object according to the structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}, also truncated and reproduced below: 

Reviewer: ```suggestion
The values of `Gas Price`, `Max Fee Per Gas` and `Max Priority Fee Per Gas` for the applicable transaction types can be read from the block JSON object according to the structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}, also truncated and reproduced below: 
```

=== "EIP1559"
    |      EVM Field       |                               Block JSON Field                               |
    |:--------------------:|:----------------------------------------------------------------------------:|

>     |     MaxFeePerGas     |     `extrinsics[extrinsic_number].args.transaction.eip1559.maxFeePerGas`     |
>     | MaxPriorityFeePerGas | `extrinsics[extrinsic_number].args.transaction.eip1559.maxPriorityFeePerGas` |

Reviewer: ```suggestion
    |     Max Fee Per Gas     |     `extrinsics[extrinsic_number].args.transaction.eip1559.maxFeePerGas`     |
    | Max Priority Fee Per Gas | `extrinsics[extrinsic_number].args.transaction.eip1559.maxPriorityFeePerGas` |
```

=== "Legacy"
    | EVM Field |                        Block JSON Field                         |
    |:---------:|:---------------------------------------------------------------:|
    | Gas Price | `extrinsics[extrinsic_number].args.transaction.legacy.gasPrice` |

=== "EIP2930"
    | EVM Field |                         Block JSON Field                         |
    |:---------:|:----------------------------------------------------------------:|
    | Gas Price | `extrinsics[extrinsic_number].args.transaction.eip2930.gasPrice` |

> The `BaseFee`, introduced in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559){target=_blank}, is a value set by the network itself. The `BaseFee` for `EIP1559` type transactions is currently static on Moonbeam networks and has the following assigned value:

Reviewer: ```suggestion
The `Base Fee`, introduced in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559){target=_blank}, is a value set by the network itself. The `Base Fee` for `EIP1559` type transactions is currently static on Moonbeam networks and has the following assigned value:
```

=== "Moonbeam"
    | Variable |  Value   |
    |:--------:|:--------:|

>     | BaseFee  | 100 Gwei |

Reviewer: ```suggestion
    | Base Fee  | 100 Gwei |
```

=== "Moonriver"
    | Variable | Value  |
    |:--------:|:------:|

>     | BaseFee  | 1 Gwei |

Reviewer: ```suggestion
    | Base Fee  | 1 Gwei |
```

=== "Moonbase Alpha"
    | Variable | Value  |
    |:--------:|:------:|

>     | BaseFee  | 1 Gwei |

Reviewer: ```suggestion
    | Base Fee  | 1 Gwei |
```
